### PR TITLE
updated the documentation for call_on_name

### PR DESCRIPTION
**Pull request by alexanderkjall**

### CODE REVIEW

File: cursive-core/src/cursive.rs [hunk 1]
```diff
@@ -521,9 +521,9 @@ impl Cursive {
         self.root.call_on(sel, callback)
     }
 
-    /// Tries to find the view identified by the given id.
+    /// Tries to find the view identified by the given name.
     ///
-    /// Convenient method to use `call_on` with a `view::Selector::Id`.
+    /// Convenient method to use `call_on` with a `view::Selector::Name`.
```
Comment (Apitronix) [inline cursive-core/src/cursive.rs:526]: Prefer mention "view::Selector::Name" as an internal link than a simple string.
You can do it using square brackets to surround the backwards simple quotes around your string. And don't forget to build the doc to ensure everything works before committing.

**Review by Apitronix — COMMENTED** [review]
Thank you for your update, there is a little thing you can do to improve it again. :)

File: cursive-core/src/cursive.rs (hunk 1 above)
Comment (alexanderkjall) [reply cursive-core/src/cursive.rs]: Thanks for the suggestion, now fixed :)

**Review by gyscos — APPROVED** [review]

**Comment by gyscos** [comment]
Thanks for the work!

**Pull request merged**